web/modules/custom/joinup_invite/joinup_invite.behat.inc

```diff
@@ -22,6 +27,69 @@ class JoinupInviteSubContext extends DrupalSubContextBase implements DrupalSubCo
 
   use NodeTrait;
   use UserTrait;
+  use RdfEntityTrait;
+
+  /**
+   * Asserts that the user has an invitation in the given group.
+   *
+   * @param string $user
+   *   The user name.
+   * @param string $status
+   *   The invitation status.
+   * @param string $label
+   *   The group label.
+   * @param string $type
+   *   The group type.
+   *
+   * @throws \Exception
+   *   Thrown when an invalid status is requested or the requested invitation
+   *   is not found.
+   *
+   * @Then :user should have a(n) :status invitation in the :label :type
+   */
+  public function userShouldBeInvitedToGroup(string $user, string $status, string $label, string $type): void {
+    $statuses = [
+      'accepted' => InvitationInterface::STATUS_ACCEPTED,
+      'pending' => InvitationInterface::STATUS_PENDING,
+    ];
+    if (!isset($statuses[$status])) {
+      throw new Exception('Only the following statuses are allowed: ' . implode(', ', array_keys($statuses)));
+    }
+    $user = $this->getUserByName($user);
+    $group = $this->getRdfEntityByLabel($label, $type);
+    $invitation_info = $this->getInvitationInfo($type);
+    $invitation = $this->loadInvitationUnchangedByEntityAndUser($group, $user, $invitation_info['invitation_bundle']);
+    if (empty($invitation)) {
+      throw new Exception("No invitation was found for {$user->getAccountName()} in the '{$group->label()}' {$group->bundle()}");
+    }
+    PHPUnit_Framework_Assert::assertEquals($invitation->getStatus(), $statuses[$status]);
+  }
+
+  /**
+   * Asserts that there is no invitation for the given user and group.
+   *
+   * @param string $user
+   *   The user name.
+   * @param string $label
+   *   The group label.
+   * @param string $type
+   *   The group type.
+   *
+   * @throws \Exception
+   *   Thrown when an invalid status is requested or the requested invitation
+   *   is not found.
+   *
+   * @Then no invitation should exist for the user :user in the :label :type
+   */
+  public function assertNoInvitation(string $user, string $label, string $type): void {
+    $user = $this->getUserByName($user);
+    $group = $this->getRdfEntityByLabel($label, $type);
+    $invitation_info = $this->getInvitationInfo($type);
+    $invitation = Invitation::loadByEntityAndUser($group, $user, $invitation_info['invitation_bundle']);
+    if (!empty($invitation)) {
+      throw new Exception("An invitation was found for {$user->getAccountName()} in the '{$group->label()}' {$group->bundle()}");
```

Comment:
Always use the full namespace when referring top-level-namespace classes. We are using `\Drupal`, not `Drupal`. So, s/`Exception`/`\Exception`.